tests/unit/test_hnsw_multi.cpp

```diff
@@ -649,10 +649,9 @@ TYPED_TEST(HNSWMultiTest, search_empty_index) {
     VecSimQueryResult_IteratorFree(it);
     VecSimQueryResult_Free(res);
 
-    // TODO: uncomment when support for HNSW Multi range is enabled
-    // res = VecSimIndex_RangeQuery(index, query, 1.0, NULL, BY_SCORE);
-    // ASSERT_EQ(VecSimQueryResult_Len(res), 0);
-    // VecSimQueryResult_Free(res);
+    res = VecSimIndex_RangeQuery(index, query, 1.0, NULL, BY_SCORE);
+    ASSERT_EQ(VecSimQueryResult_Len(res), 0);
+    VecSimQueryResult_Free(res);
 
     // Add some vectors and remove them all from index, so it will be empty again.
```

Comment:
do we have a test for 0 results because everything is out of range?

Reply:
We might have/want one on `test_hnsw.cpp`, it's not related to multi-value.